### PR TITLE
Enables Header creation on all projects

### PR DESCRIPTION
**Pull request by ignasi35**
Compilation and test should now add copyright header automatically.

### CODE REVIEW

**Comment by ignasi35** [comment]
Meh! Travis failed because it run out of Disk space:

```
! @767ln6h5h - Internal server error, for (POST) [/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[IOException: out of disk space]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:255)
Caused by: java.io.IOException: out of disk space
	at com.google.common.jimfs.HeapDisk.allocate(HeapDisk.java:121)
[error]   x work if there's no crlf at the start
```

in `test-scala-211` and `test-scala-212`.

**Comment by ignasi35** [comment]
Oh, wait, not true. 

That "Disk Out of space" is expected.

**Comment by ignasi35** [comment]
The actual error is:


```
[error]   x work if there's no crlf at the start
[error]    Right(MultipartFormData(Map(text2:colon -> Vector(the second text field), noQuotesText1:colon -> Vector(text field with unquoted name and colon), text1 -> Vector(the first text field), noQuotesText1 -> Vector(text field with unquoted name)),Vector(FilePart(file1,file1.txt,Some(text/plain),play.api.libs.Files$DefaultTemporaryFileCreator$DefaultTemporaryFile@4e0aae44), FilePart(file2,file2.txt,Some(text/plain),play.api.libs.Files$DefaultTemporaryFileCreator$DefaultTemporaryFile@1144391f)),Vector())) is Right but Some(FilePart(file1,file1.txt,Some(text/plain),play.api.libs.Files$DefaultTemporaryFileCreator$DefaultTemporaryFile@4e0aae44)) is Some but java.nio.file.NoSuchFileException: /tmp/playtemp6616104135362630019/multipartBody4331674611954385814asTemporaryFile (MultipartFormDataParserSpec.scala:58)
```

I can't reproduce it locally. :-(

**Comment by ignasi35** [comment]
Hmmm, `testOnly   play.it.http.parsing.MultipartFormDataParserSpec` seems to be flaky (even locally).

**Comment by gmethvin** [comment]
Might be simpler to add this to `PlayCrossBuiltProject` and the other helpers rather than to each project individually.

**Comment by ignasi35** [comment]
> Might be simpler to add this to PlayCrossBuiltProject and the other helpers rather than to each project individually.

thanks @gmethvin

**Comment by ignasi35** [comment]
@gmethvin I went ahead and added the plugin in all:

```
 PlayCrossBuiltProject
 PlayDevelopmentProject
 PlayNonCrossBuiltProject
 PlaySbtPluginProject
 PlaySbtProject
```

File: framework/build.sbt [hunk 1]
```diff
@@ -170,6 +172,7 @@ lazy val PlayDocsProject = PlayCrossBuiltProject("Play-Docs", "play-docs")
     ).dependsOn(PlayAkkaHttpServerProject)
 
 lazy val PlayGuiceProject = PlayCrossBuiltProject("Play-Guice", "play-guice")
+
```
Comment (gmethvin) [inline framework/build.sbt]: Empty lines?

**Comment by marcospereira** [comment]
Backport to 2.6.x: c79faeb0439224990f2deb6e9ba60eb8cc76067f

**Pull request merged**